config/locales/en.yml

```diff
@@ -64,7 +64,7 @@ en:
     error:
       generic: |
         Oops! Something went wrong and we couldn't process that action. You
-        might want to try it again, or if that doesn't work, refresh the page.
+        might want to try it again.
```

Comment:
Ah, I see. That's a bit worrisome, since there might be error conditions where the only way to recover is in fact to refresh the page (I wouldn't expect this to be common, but it's the only 100% effective escape hatch we have). How would we handle this in an app context? Close and re-launch the app?